src/React.MSBuild/TransformJsx.cs

```diff
@@ -52,7 +52,8 @@ public override bool Execute()
 			config
 				.SetReuseJavaScriptEngines(false)
 				.SetUseHarmony(UseHarmony)
-				.SetStripTypes(StripTypes);
+				.SetStripTypes(StripTypes)
+				.AddScript(string.Empty);
```

Comment:
I had to add an item here because without it the MSBuild step would blow up.  I am not too familiar with what this build step is doing, but by making the loading of the JSX Transform conditional on having at least one Script caused this to blow up unless this was added.  If you can think of a more elegant solution that would be great!